content/docs/integration/integrate-with-apps@ko.md

```diff
@@ -0,0 +1,195 @@
+---
+$title: 앱과 AMP 통합 
+---
+
+이 가이드는 AMP 페이지를 통합하고 연결하려는 모바일 및 웹 앱 개발자를 위해 작성되었습니다. AMP 버전의 공유 URL을 로드하여 더 빠른 사용자 환경을 제공하는 모바일 채팅 앱을 예로 들어 보겠습니다.
+
+[TOC]
+
+## 링크를 AMP로 변환  
+
+AMP를 사용하면 기본 앱 또는 모바일 웹 앱 내에서 외부 웹사이트를
+거의 즉각적으로 렌더링할 수 있습니다. 콘텐츠의 URL을 상응하는
+AMP URL(있는 경우)과 일치시키고, 원래 버전 대신 AMP 버전을
```

Comment:
- Current: "AMP URL(있는 경우)과 일치시키고"
- Proposed: "AMP URL(있는 경우)로 대치하고"
- Justification: the current translation "일치시키고" is a literal translation for "matching". But the meaning of the original text is "to use AMP URL instead of the original URL" or "to replace the original URL with AMP URL". In that case, "대치하고" conveys the meaning.